dysnix/proxysql/templates/daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           command: ["proxysql", "-c", "/etc/proxysql/proxysql.cnf", "-f"]
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: {{ template "proxysql.image" . }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
Thank you for the contribution @reixd !
Would you please consider getting back the template and rather fixing it. It supports registry injection, whereas the above code does not.

Reply:
You can still specify a different registry, just need to set something like:
`image.repository = "hub.foo.bar.com/proxysql/proxysql"`